packages/contentstack-branches/src/utils/interactive.ts

```diff
@@ -247,3 +271,66 @@ export async function askBranchNameConfirmation(): Promise<string> {
     validate: inquireRequireFieldValidation,
   });
 }
+
+export async function selectContentMergeCustomPreferences(payload) {
+  // parse rows
+  const tableRows = [];
+  if (payload.modified?.length || payload.added?.length) {
+    forEach(payload.added, (item: BranchDiffRes) => {
+      const row: any = {};
+      row.name = `+ ${item.title}`;
+      row.status = 'added';
+      row.value = item;
+      tableRows.push(row);
+    });
+
+    forEach(payload.modified, (item: BranchDiffRes) => {
+      const row: any = {};
+      row.name = `± ${item.title}`;
+      row.status = 'modified';
+      row.value = item;
+      tableRows.push(row);
+    });
```

Comment:
All code appears identical except for the status. To mitigate redundancy, incorporating an inline function might be beneficial.